teamB/seongmin/westagram/app.js

```diff
@@ -57,9 +103,87 @@ app.post('/users/sign-up', async(req, res) => {
         [name, email, profile_image, password]
     );
 
-    res.status(201).json({ mesaage: "userCreated"});
+    res.status(201).json({ mesaage: "userCreated" });
 })
 
+app.post('/posts', async (req, res) => {
+    const { title, content, user_id } = req.body;
+
+    const userCheck = await dataSource.query(`
+    SELECT
+        count(*) 
+    FROM users 
+    WHERE id = ${user_id}
+    IS TRUE`); //게시글 작성 유저 확인
+
+    if (userCheck[0]['count(*)'] == 1) {
+        await dataSource.query(
+            `INSERT INTO posts (
+                title,
+                content,
+                user_id
+            ) VALUES (?, ?, ?);`, [title, content, user_id]);
+        return res.status(201).json({ message: "postCreated" });
+    }
+
+    res.status(404).json({ message: "user id is not exist!" });
+});
+
+app.post('/likes/:postId/:userId', async (req, res) => {
+    const postId = req.params.postId;
+    const userId = req.params.userId;
```

Comment:
구조분해 할당으로 한 줄로 줄일 수 있을 것 같습니다.
```suggestion
    const { userId, postId } = req.params;
```

Reply:
구조분해 할당 방식으로 코드 정리하였습니다.